DiscordChatExporter.Cli/Commands/Base/ExportCommandBase.cs

```diff
@@ -69,6 +69,34 @@ protected async ValueTask ExportAsync(IConsole console, Guild guild, Channel cha
             console.Output.WriteLine("Done.");
         }
 
+        public void ExecuteAsync()
+        {
+            if (ShouldReuseMedia && !ShouldDownloadMedia)
+            {
+                var shouldDownloadMediaOptionName = GetAttributeName(nameof(ShouldDownloadMedia));
+                var shouldReuseMediaOptionName = GetAttributeName(nameof(ShouldReuseMedia));
+
+                Console.WriteLine($"The \"--{shouldReuseMediaOptionName}\" option cannot be used without the \"--{shouldDownloadMediaOptionName}\" option.");
+                Environment.Exit(1);
+            }
+
+            string GetAttributeName(string name)
+            {
+                var property = typeof(ExportCommandBase).GetProperty(name);
+                if (property == null) throw new Exception("Could not find property: " + name);
+
+                var attributes = property.GetCustomAttributes(true);
+                foreach (var attribute in attributes)
+                {
+                    var commandOptionAttribute = attribute as CommandOptionAttribute;
+                    if (commandOptionAttribute != null && commandOptionAttribute.Name != null)
+                    {
+                        return commandOptionAttribute.Name;
+                    }
+                }
+                throw new Exception("Unable to find name of property");
+            }
```

Comment:
It's fine to use hardcoded values. If we really wanted to, we could use `const string MediaOptionName` and then use it in both places, but I don't think it's necessary as option names are not going to change since it's a breaking change.

Reply:
[I think attribute arguments have to be constant expressions.](https://stackoverflow.com/questions/16710533/passing-static-array-in-attribute), so we can't have a `MediaOptionName` variable and pass it into the `CommandOption`. We could just use the hard-coded `"reuse-media"` twice in the code, though.